python/taichi/lang/matrix.py

```diff
@@ -925,7 +925,7 @@ def field(cls,
             name (string, optional): The custom name of the field.
             offset (Union[int, tuple of int], optional): The coordinate offset of all elements in a field.
             needs_grad (bool, optional): Whether the Matrix need gradients.
-            layout (:class:`~taichi.lang.impl.Layout`, optional): The field layout, i.e., Array Of Structure(AOS) or Structure Of Array(SOA).
+            layout (Layout, optional): The field layout, i.e., Array Of Structure (AOS) or Structure Of Array (SOA).
```

Comment:
If `:class:` still works, it might be good to keep it since it'll link to Layout docstring on the website?

Reply:
well type annotations in the parentheses can be automatically linked..